pandasai/llm/gpt4all.py

```diff
@@ -0,0 +1,181 @@
+"""GPT4All LLMs
+This module provides a family of commercially / non-commercially available 
+LLMs maintained by Nomic AI. With GPT4All we can easily download models like 
+Falcon, Llama etc and run them seamlessly using CPU with lower RAMs. 
+
+Example:
+    Use below example to call GPT4All supporrted models
+
+    >>> from pandasai.llm.gpt4all import GPT4AllLLM
+"""
+import os
+import requests
+from typing import Optional
+
+from pandasai.prompts.base import Prompt
+from .base import LLM
+
```

Comment:
```suggestion

logger = logging.getLogger(__name__)
```

Reply:
Sure that works, and thanks I  learned something new here.